core/src/main/java/smile/clustering/CLARANS.java

```diff
@@ -230,7 +230,7 @@ private double getRandomNeighbor(T[] data, T[] medoids, int[] y, double[] d) {
         int n = data.length;
 
         int index = Math.randomInt(k);
-        T medoid = null;
+        T medoid;
```

Comment:
This is actually to remove warning like "uninitialized variable". Maybe today javac doesn't report that warning any more?